queries/contributions.sql

```diff
@@ -0,0 +1,61 @@
+
+-- name: getContributionCountByType :many
+SELECT type, count(*) as count
+FROM contributions
+WHERE id = ANY (@ids::int[])
+group by type;
```

Comment:
This one is a bit of a headscratcher for me.

Reply:
lol. its just a count by a set of ids, because im getting ids from getContributionIDsForPersonsWithRoles.
i could include the type in that one and do the counting with go, but this was easier to wrap my head around than the relationloader stuff

Reply:
Ok, but this is not very bulk loader friendly, is it? It will have to run for each contributor separately if I understand it correctly.

Reply:
yes, true, good point. I dont think I should spend time to fix it now though, since we are not listing types in lists anyway. It's only used on the contributor page.